src/components/controls/TextArea/__stories__/TextAreaShowcase.tsx

```diff
@@ -84,3 +84,47 @@ export function TextAreaShowcase() {
         </div>
     );
 }
+
+export function TextAreaCustomShowcase() {
+    const [value, setValue] = React.useState('');
+    // const [isErrorMessageVisible, setErrorMessageVisibility] = React.useState(false);
+
+    const textAreaProps: TextAreaProps = {
+        className: b('text-area'),
+        onUpdate: setValue,
+        value,
+        placeholder: 'placeholder',
+    };
+    return (
+        <div className={b('text-area-examples')}>
+            <div>
+                <h3 className={b('section-header')}>Normal</h3>
+                <TextArea
+                    {...textAreaProps}
+                    value={`
+multi
+line
+value`.trim()}
```

Comment:
Here and below. I suggest u to use `defaultValue` here, otherwise controls with hardcoded value have a really weird behaviour